app/views/personal_details/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "shared/form_errors_summary", errors: @personal_details.errors %>
```

Comment:
An unfortunate side effect of doing things the correct way is that this form is now duplicated! Could it be a partial, perhaps?

Reply:
yep, good point.